src/property.ts

```diff
@@ -32,6 +36,8 @@ export function hasNestedProperty(prop: Property) {
     case Property.TYPE:
     case Property.BIN_MAXBINS:
     case Property.SCALE_TYPE:
+    case Property.TRANSFORM_CALCULATE:
```

Comment:
I would say remove all of transform for now.  That's not the point of this issue and you're not doing it correctly anyway.